traveltimepy/accept_type.py

```diff
@@ -0,0 +1,8 @@
+from enum import Enum
+
+
+class AcceptType(Enum):
+    JSON = 'application/json'
+    BOUNDING_BOXES_JSON = 'application/vnd.bounding-boxes+json'
+    GEO_JSON = 'application/geo+json'
+    OCTET_STREAM = 'application/octet-stream'
```

Comment:
If this is supposed to reflect [this](https://docs.traveltime.com/api/reference/isochrones#Response-Body)
then it's missing few options

Reply:
They are not done yet. It will be the second pr

Reply:
When they'll be done add example to readme on how to call time map with geojson option too

Reply:
yeah, of course. This pr is huge, so I didn't want to make it bigger.